src/transaction/log_manager.c

```diff
@@ -11213,7 +11248,9 @@ cdc_get_recdes (THREAD_ENTRY * thread_p, LOG_LSA * undo_lsa, RECDES * undo_recde
   int tmpbuf_index;
 
   char *log_pgbuf[IO_MAX_PAGE_SIZE + MAX_ALIGNMENT];
-  LOG_PAGE *log_page_p = NULL;
+  char *preserved_log_pgbuf[IO_MAX_PAGE_SIZE + MAX_ALIGNMENT];
```

Comment:
Done. LOG_PAGESIZE is right. 

When developing the CDC module, I used IO_MAX_PAGE_SIZE as the constant to represent the page size throughout the module. This was because the log manager module also predominantly used IO_MAX_PAGE_SIZE, so I followed its lead when developing the CDC module.

I think, changing the constant for page size to `LOG_PAGESIZE` throughout the CDC module would need to be addressed in a separate issue. For now, I think it would be better to modify the constants in the functions touched by this PR. What do you think?

Reply:
@joohok I understand why IO_MAX_PAGE_SIZE is used. If it is, I recommend just leaving it as is or handling it all together in another issue.

Reply:
OK, I will make another issue